sdcm/cluster_aws.py

```diff
@@ -339,6 +349,11 @@ def restart(self):
                        self._instance.public_ip_address)
         self.remoter.hostname = self._instance.public_ip_address
 
+        if any(ss in self._instance.instance_type for ss in ['i3', 'i2']):
+            self.remoter.run('sudo /usr/lib/scylla/scylla-ami/scylla_create_devices')
```

Comment:
Is this script add a record to /etc/fstab?

Reply:
scylla_create_devices script will run scylla_raid_setup with parameter `--update-fstab`
The /etc/fstab file will be updated.